source/main/utils/processes.ts

```diff
@@ -49,3 +63,39 @@ export const getProcess = async (
     return null;
   }
 };
+
+/**
+ * Like `find from 'find-process'`, but wraps the execution into setting locale to C (available anywhere).
+ *
+ * Inside Nix chroot for Linux installation, it is possible there will not be locale definitions for user’s LC_ALL,
+ * which variable is inherited by the chroot from user’s regular system.
+ *
+ * Unfortunately, `/bin/sh` inside the Nix chroot outputs warnings to `stderr` about missing locales then.
+ *
+ * Then, `find-process` interprets non-empty stderr as some error, and throws.
+ *
+ * Another solution would be to patch `find-process` to construct the `ps` invocation itself, not using `/bin/sh`
+ * for that (i.e. `exec('ps', 'aux')` instead of `exec('ps aux')` which is really `exec('/bin/sh', '-c', 'ps aux')`.
```

Comment:
That’s not a particularly amazing solution, but I decided it’s quicker than patching upstream…